src/ucs/datastruct/arbiter.c

```diff
@@ -174,6 +171,25 @@ void ucs_arbiter_group_purge(ucs_arbiter_t *arbiter,
     }
 }
 
+size_t ucs_arbiter_group_num_elems(ucs_arbiter_group_t *group)
+{
+    ucs_arbiter_elem_t *elem = group->tail;
+    size_t num_elems;
+
+    if (elem == NULL) {
```

Comment:
can use `ucs_arbiter_group_is_empty()` instead

Reply:
this call is used in tests

Reply:
> this call is used in tests

ah, didn't notice this